packages/stark-ui/assets/styles/_base.scss

```diff
@@ -99,3 +99,190 @@ body {
 .flex-fill {
   flex: 1;
 }
+
+/* stark-layout */
+$stark-layout-spacing: 15px;
+.stark-layout {
+  display: flex;
```

Comment:
`.stark-layout` (the flexbox part) is replaced by Angular Flex-Layout. I added documentation with #1086